sparkbox/apprentices-page-next
# chore: add stylelint

Adds stylelint and fixes linting problems. Adds some commands to run the
linters.

References FSA21V2-144

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci` && `npm run lint`.
